.github/workflows/ci.yml

```diff
@@ -9,13 +9,13 @@ jobs:
     name: sanity
     runs-on: ubuntu-22.04
     steps:
-    - uses: actions/setup-go@v3
-      with:
-        go-version: 1.19
-      id: go
     - uses: actions/checkout@v3
       with:
         fetch-depth: 0
+    - uses: actions/setup-go@v4
+      with:
+        go-version-file: go.mod
+      id: go
     - id: go-cache-paths
```

Comment:
If I'm not mistaken, I think setup-go@v4 takes care of all the caching now?

Reply:
v4 does do cacheing by default now, cache steps have been removed.